src/XamlStyler/MarkupExtensions/Parser/MarkupExtensionParser.cs

```diff
@@ -1,5 +1,6 @@
-// � Xavalon. All rights reserved.
+// � Xavalon. All rights reserved.
```

Comment:
Where are these changes coming from for these last three files anyway? Can we revert?

Reply:
Needed to add two `using` statements along the way, seems when using `#if DEBUG` these were breaking compilation, so added them.